apps/src/studio/levels.js

```diff
@@ -1750,14 +1750,14 @@ levels.js_hoc2015_event_two_items = {
   "editCode": true,
   "background": "forest",
   "wallMap": "blank",
-  "softButtons": ["downButton", "upButton"],
+  "softButtons": ["leftButton", "rightButton"],
   "codeFunctions": {
     "moveRight": null,
     "moveLeft": null,
-    "moveUp": null,
-    "moveDown": null,
+    "whenLeft": null,
+    "whenRight": null
   },
-  "startBlocks": "",
+  "startBlocks": "function whenLeft() {\n  \n}\nfunction whenRight() {\n  \n}",
```

Comment:
Might make it easier for you to parse/maintain if you stick line breaks in there (current approach is also fine if it works for you), i.e.
```
"startBlocks": "" +
  "function whenLeft() {\n" +
  "  \n" +
  "}\n"
  "function whenRight() {\n" +
  "  \n" +
  "}\n"
```
or
```
"startBlocks": [
  "function whenLeft() {",
  "  ",
  "}",
  "function whenRight() {",
  "  ",
  "}"].join('\n')